src/zkevm_specs/evm/instruction.py

```diff
@@ -368,6 +377,42 @@ def range_check(self, value: Expression, n_bytes: int) -> bytes:
         except OverflowError:
             raise ConstraintUnsatFailure(f"Value {value} has too many bytes to fit {n_bytes} bytes")
 
+    def constrain_abs_word(self, x: RLC, x_abs: RLC, x_is_neg: FQ):
+        expected_abs, expected_is_neg = self.abs_word(x)
+        self.constrain_equal(self.word_is_equal(x_abs, expected_abs), FQ(1))
+        self.constrain_equal(x_is_neg, expected_is_neg)
+
+    # Return a tuple of `abs(x)` and `x_is_neg`.
```

Comment:
Add extra comment for the case `x == -2^255`.

Reply:
Added comment for this function `abs_word`:
```
# Return a tuple of `abs(x)` and `x_is_neg`. For a special case when
# `x = -(1 << 255)`, this function returns the ABS value of `-(1 << 255)`,
# since it is sign overflow.
```